lib/python/picongpu/plugins/jupyter_widgets/phase_space_visualizer.py

```diff
@@ -0,0 +1,58 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017-2018 PIConGPU contributors
+Authors: Sebastian Starke
+License: GPLv3+
+"""
+
+from .base_visualizer import Visualizer as BaseWidgetVisualizer
+from picongpu.plugins.plot_mpl.phase_space_visualizer import Visualizer\
+     as PICVisualizer
+
+from ipywidgets import widgets
+
+
+class Visualizer(BaseWidgetVisualizer):
```

Comment:
I would like to call this `PhaseSpaceWidget`, etc.
```suggestion
class PhaseSpaceWidget(BaseWidget):
```